bucket/compactor.json

```diff
@@ -0,0 +1,38 @@
+{
+    "version": "0.10.1",
+    "description": "A user interface for Windows 10 filesystem compression",
+    "homepage": "https://github.com/Freaky/Compactor",
+    "license": "MIT",
+    "architecture": {
+        "64bit":{
+            "url": "https://github.com/Freaky/Compactor/releases/download/v0.10.1/Compactor-0.10.1.7z",
+            "hash": "sha256:1dc75fd5dad212e0cbf29a363632e0299945183d722b2c00e527e54ed86de379",
+            "extract_dir": "Compactor-0.10.1"
+        },
+        "32bit":{
+            "url": "https://github.com/Freaky/Compactor/releases/download/v0.10.1/Compactor-0.10.1-i686.7z",
+            "hash": "sha256:015e358422db0eeb4d71305aaa4a74991f43c46074e26d6a9487645ab109083b",
+            "extract_dir": "Compactor-0.10.1-i686"
+        }
+    },
+    "bin": "Compactor.exe",
+    "shortcuts": [
+        [
+            "Compactor.exe",
+            "Compactor"
+        ]
+    ],
+    "checkver": {
+        "github": "https://github.com/Freaky/Compactor"
+    },
+    "autoupdate": {
+        "architecture": {
+            "64bit":{
+                "url": "https://github.com/Freaky/Compactor/releases/download/v$ver/Compactor-$ver.7z"
```

Comment:
```suggestion
                "url": "https://github.com/Freaky/Compactor/releases/download/v$version/Compactor-$version.7z",
                "extract_dir": "Compactor-$version"
```

Reply:
any difference between $ver and $version?

Reply:
$ver is not valid